mappings/none/zb.mapping

```diff
@@ -1,5 +0,0 @@
-CLASS none/zb
```

Comment:
Well this becomes `EntitySkeletonBase` but git showed this as deleted instead of renamed.